stdlib/unicodedata.pyi

```diff
@@ -17,7 +17,10 @@ def combining(__chr: str) -> int: ...
 def decimal(__chr: str, __default: _T = ...) -> int | _T: ...
 def decomposition(__chr: str) -> str: ...
 def digit(__chr: str, __default: _T = ...) -> int | _T: ...
-def east_asian_width(__chr: str) -> str: ...
+
+EastAsianWidth: TypeAlias = Literal["F", "H", "W", "Na", "A", "N"]
```

Comment:
It needs to have a name starting with a leading underscore, or stubtest will complain about the object not existing at runtime :)

```suggestion
_EastAsianWidth: TypeAlias = Literal["F", "H", "W", "Na", "A", "N"]
```

Reply:
Thank you for guiding me.  I amended the commit.  Could you give it a look again?